### ISSUE TITLE
Would be great to let us select Commerce product types

### ISSUE THREAD

**Issue by echantigny** [page-8/craft-manytomany#16]
I really like the idea of this field type, but it would be great to let us use it with Commerce Product.  I'm adding Categories to my products and would like to see all of the associated products when I go see my Categories entry page.

**Comment by fantasticmachine** [comment]
+1 for this

**Comment by peter-tell** [comment]
Yeah, another good feature request. We've long wanted to implement all Element Types. Hopefully we can carve out some time to look into it. I'm also open to anyone forking as well.
